src/core/utils/types/reservation-type.ts

```diff
@@ -8,4 +8,5 @@ export interface ReservationType extends ListType {
   state: string;
   pickupBranch?: string;
   expectedRedeemDateUtc?: string;
+  pickupNumber?: string;
```

Comment:
Could we make `pickupNumber` a number instead of a string? Based on the naming that seems like the right type for the value.

Reply:
It just isn't a number, it is a string, like "Reserveringshylde 74". I use the name from the [ReservationDetailsV2](https://github.com/danskernesdigitalebibliotek/dpl-react/blob/release/4/src/core/fbs/model/reservationDetailsV2.ts) to be consistent... But I agree it is a misleading name... Should I change it?

Reply:
Good question. I think it is fair to keep it around for consistency but adding a comment explaining why it is a string, perhaps with a example of what it might contain, would be helpful.